healthz/config/thresholds.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"strconv"
+	"sort"
+	"strings"
+
+	"github.com/gravitational/trace"
+)
+
+const DefaultThreshold = 10
+
+type Thresholds struct {
+	NodeCounts []uint
+	DeadPcts   map[uint]uint8
+}
+
+type uintSlice []uint
+
+func (t uintSlice) Len() int {
+	return len(t)
+}
+
+func (t uintSlice) Swap(i, j int) {
+	t[i], t[j] = t[j], t[i]
+}
+
+func (t uintSlice) Less(i, j int) bool {
+	return t[i] < t[j]
+}
+
+func NewThresholds(unparsed string) (*Thresholds, error) {
+	t := Thresholds{}
+	t.DeadPcts = make(map[uint]uint8)
+	for _, unparsedThresh := range strings.Split(unparsed, ",") {
+		a := strings.Split(unparsedThresh, ":")
+		if len(a) != 2 {
+			return nil, trace.Errorf("Unable to parse")
```

Comment:
Like Sasha suggested - move this to configuration for easier handling.
Otherwise, provide enough context in the error to explain what was wrong with input.